config/v1/types_infrastructure.go

```diff
@@ -457,6 +462,32 @@ type AWSResourceTag struct {
 	Value string `json:"value"`
 }
 
+type ClusterDNSConfig struct {
+	// APIServerDNSConfig contains DNS configuration for the API Server.
+	// +optional
+	APIServerDNSConfig []DNSConfig `json:"apiServerDNSConfig,omitempty"`
+
+	// InternalAPIServerDNSConfig contains DNS configuration for the Internal API Server.
```

Comment:
What happens when this is omitted? What's the expectation?

Reply:
Same as the DNS config for API server. See https://github.com/openshift/api/pull/1397#discussion_r1131541743